selfdrive/car/honda/carcontroller.py

```diff
@@ -74,6 +74,10 @@ def brake_pump_hysteresis(apply_brake, apply_brake_last, last_pump_ts, ts):
   if ts - last_pump_ts < 0.2 and apply_brake > 0:
     pump_on = True
 
+  # some hybrid nidecs like Odyssey hybrid and Clarity need to keep pump on during braking or they'll lose pressure
+  if self.CP.carFingerprint == CAR.ODYSSEY_HYBRID:
```

Comment:
this looks not necessary?

pump_on is always set to True for ODYSSEY_HYBRID AFTER the brake_pump_hysteresis?